tests/library/selectors-register.spec.ts

```diff
@@ -145,6 +145,11 @@ it('should handle errors', async ({ playwright, browser }) => {
   await page.close();
 });
 
+it('should throw "already registered" error when registering', { annotation: { type: 'issue', description: 'https://github.com/microsoft/playwright/issues/36467' } }, async ({ playwright }) => {
+  await playwright.selectors.register('foo', createTagSelector);
```

Comment:
Choose a unique name, e.g. `alreadyRegistered`, otherwise this will conflict with other some other test.